tests/foreman/performance/common/perf_pulp.py

```diff
@@ -0,0 +1,151 @@
+"""Test utilities for writing Pulp tests
+
+Part of functionalities of Pulp are defined in this module
+and have utilities of single repository synchronization, single
+sequential repository sync, sequential repository re-sync.
+
+"""
+
+
+from robottelo.cli.repository import Repository
+
+from perf_standup import StandupPerfTest
+import logging
+
+
+class PulpPerfTest:
+    """Performance Measurement of RH Satellite 6
+
+    Pulp Synchronization functionality
+    """
+
+    # parameters
+    org_id = 1
+
+    logger = logging.getLogger('robottelo')
+    logger.setLevel(logging.DEBUG)
```

Comment:
Please do not set the level, it will be already set according to the properties config.